happybase/pool.py

```diff
@@ -72,11 +76,12 @@ def __init__(self, size, **kwargs):
             connection = Connection(**connection_kwargs)
             self._queue.put(connection)
 
-        # The first connection is made immediately so that trivial
-        # mistakes like unresolvable host names are raised immediately.
-        # Subsequent connections are connected lazily.
-        with self.connection():
-            pass
+        if autoconnect:
+            # The first connection is made immediately so that trivial
+            # mistakes like unresolvable host names are raised immediately.
+            # Subsequent connections are connected lazily.
```

Comment:
nitpick: can you change this into

"Immediately open one connection so that trivial mistakes like unresolvable host names raise an exception upon pool instantiation instead of on first use. Subsequent connections are connected lazily."